tracee/pipeline.go

```diff
@@ -106,6 +108,36 @@ func (t *Tracee) processRawEvent(done <-chan struct{}, in <-chan RawEvent) (<-ch
 	return out, errc, nil
 }
 
+func (t *Tracee) getStackTrace(StackID [4]byte) (string, error) {
+	stackFrameSize := (strconv.IntSize / 8)
```

Comment:
Maybe I'm missing something, but shouldn't address size be 64bit on a 64bit system (and strconv.IntSize is 32bit, right?)

Reply:
So I think it might be poorly named but on 64bit, `strconv.IntSize` returns `64`: https://play.golang.org/p/BNHTlmWD5Dg

Reply:
I'm no go expert, I took the advice from stack overflow 😅  https://stackoverflow.com/questions/25741841/how-can-i-check-if-my-program-is-compiled-for-32-or-64-bit-processor